modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc

```diff
@@ -0,0 +1,204 @@
+// TELCODOCS-213 4.8 Using Assisted Installer to Install & Manage ocp Clusters on Bare Metal
+//
+//
+//
+
+[id=ztp-ai-install-ocp-clusters-on-bare-metal_{context}"]
+
+.Enabling Assisted Installer Service on Bare Metal
+
+The Assisted Installer Service (AIS) deploys {product-title} clusters. Red Hat Advanced Cluster Management (RHACM) 2.3+ ships with AIS, which is deployed when the MultiClusterHub Operator is enabled on the RHACM hub cluster.
+
+RHACM 2.3+ only supports *Single Node OpenShift (SNO)* cluster deployments. A SNO cluster runs OpenShift Container Platform (ocp) on top of a single BMO node. When the host is provisoned, the single node cluster acts as both supervisor and worker node.
+
+.Enable assisted installer service
+
+.Prerequisites
+
+. OpenShift Container Platform 4.8+\{product-version}
+. https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.2/html/install/installing#installing-red-hat-advanced-cluster-management-from-the-operatorhub[RHACM 2.3+ with MultiClusterHub resource created]
+. `PersistentVolume` CRs for database and file system storage
+. Install `openshift-install` [cli tool] for the desired version of OCP
```

Comment:
I don't think OCP is valid as a contraction for OpenShift prod name. Try: "Install `openshift-install` CLI for the desired version of {product-title}."